components/project-card/meta/get-involved.jsx

```diff
@@ -46,15 +46,11 @@ class GetInvolved extends React.Component {
   renderHelpLabels() {
     if (!this.props.helpTypes) return null;
 
-    let classes = classNames(`btn btn-tag`, {
-      "btn-xs": this.props.onModerationMode
```

Comment:
this `btn-xs` class never worked and wasn't something relected on the mockup either